File: Heat/ViewModels/Customers/EnableCustomerViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Customers
    Public Class EnableCustomerViewModel
        <Key> _
        Property ID As Integer

        <Display(name:="Cliente")> _
        Property CustomerName As String

        <Display(name:="Data di disabilitazione")> _
        Property DisableDate As DateTime

    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Customers/EnableCustomerViewModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Customers
{
    public class EnableCustomerViewModel
	{
		[Key()]
		public int ID { get; set; }

		[Display(Name = "Cliente")]
		public string CustomerName { get; set; }

		[Display(Name = "Data di disabilitazione")]
		public DateTime? DisableDate { get; set; }

	}
}

